### ISSUE TITLE
Writing a dataframe to a csv

### ISSUE THREAD

**Issue by ghost** [XamNalpak/xampy#3]
**Is your feature request related to a problem? Please describe.**
be able to download the output of a csv file to your current directory

**Describe the solution you'd like**
take in the dataframe and output a csv

**Describe alternatives you've considered**
NA
**Additional context**
NA

**Comment by sarathsridharababu** [comment]
Hello, from which function the output returns in dataframe format? Can you link the function here?

**Comment by ghost** [comment]
> Hello, from which function the output returns in dataframe format? Can you link the function here?

@sarathsridharababu 
was thinking of making a function that takes in a dataframe as a parameter like so

```
def returnCSV(dataframe):
     pd.to_csv(dataframe)
```

**Comment by sarathsridharababu** [comment]
Yes, we can do that or we can directly convert the dataframe into csv without writing a new function.  Can you tag the function that generates the dataframe? So that it will be easy to understand the context and the better way to implment.

**Comment by ghost** [comment]
@sarathsridharababu im not understand your point of view, please give me a code example,

The makeData() function is the one that turns a csv into a df,

Maybe you could add a parameter to the function to either read or write.
